docs/detections/alerts-view-details.asciidoc

```diff
@@ -6,111 +6,142 @@
+* **Alert reason**: Describes the source event that generated the alert. Event details are displayed in plain text and ordered logically to provide context for the alert. Click **Show full reason** to display the alert reason in the event rendered format within the <<preview-panel>>.  
 +
-[role="screenshot"]
-image::images/enriched-data-info-icon.png[Informational message on enriched data, 600]
+NOTE: The event renderer only displays if an event renderer exists for the alert type. Fields are interactive; hover over them to access the available actions.
+
+* **MITRE ATT&CK**: Provides relevant https://attack.mitre.org/[MITRE ATT&CK] framework tactics, techniques, and sub-techniques.
 
 [discrete]
-[[alert-details-insights]]
-==== Insights on alerts
+[[investigation-section]]
+== Investigation 
 
-The Insights section shows you how an alert is related to other alerts and offers ways to investigate related alerts. You can use this information to quickly find patterns between alerts and then take action.
+The Investigation section in the right panel has the following information:
 
-Within the Insights section, you can click on the title for each insight to expand or collapse it.
+* **Investigation guide**: A interactive investigation guide displays if the rule associated with the alert has an investigation guide. 
++
+TIP: To add an investigation guide to a custom rule, <<edit-rules-settings,modify the rule's advanced settings>>. 
 
-[role="screenshot"]
-image::images/insights-section.png[Insights section in Alert details flyout, 600]
+* **Highlighted fields**: Shows relevant fields for the alert and any custom highlighted fields you added to the rule. 
+//link to custom highlighted fields docs
 
-The Insights section provides the following details:
+[discrete]
+[[visualizations-section]]
+== Visualizations 
 
-* *Suppressed alerts* - Indicates that the alert was created with <<alert-suppression,alert suppression>>, and shows how many duplicate alerts were suppressed. This section only appears if alert suppression is enabled for the rule.
-* *Cases related to the alert* - Shows the total number and names of cases to which the alert has been added. Click a case's name to open its details.
-* *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.
+The Visualizations section in the right panel offers a glimpse of the processes that led up to the alert and occurred after it. Click **Visualizations** to display the following sub-sections:
 
-If you have a https://www.elastic.co/pricing[Platinum or higher subscription], these details are also included:
+* **Session view preview**: Shows a preview of <<session-view,session view>> data. Click **Session viewer preview** to open the **Session View** tab in Timeline. 
```

Comment:
@PhilippeOberti @paulewing what does the preview show, when one is available? I'm still not sure how to describe it in the docs.

Reply:
Resolving since the PR was approved during the tech review.